Add example to remove the website title (#633)
TYPO3-Documentation/TYPO3CMS-Reference-Typoscript: Documentation/Setup/Config/Index.rst

            config.pageTitleSeparator = *
            config.pageTitleSeparator.noTrimWrap = |||
            

>          If you want to get rid of the web page title, you must choose a separator that is not included in the web page title. 

Reviewer: ```suggestion
         If you want to remove web page title from the displayed title, choose a separator that is not included in the web page title. 
```

>          You then split the title from that character and return everything after it:

Reviewer: ```suggestion
         Then split the title from that character and return the second part only:
```
